OSOE-954: Update All packages

sarahelsaig opened the pull request:
[OSOE-954](https://lombiq.atlassian.net/browse/OSOE-954)

[OSOE-954]: https://lombiq.atlassian.net/browse/OSOE-954?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

sarahelsaig committed:
submodules

sarahelsaig committed:
Update All packages

sarahelsaig committed:
submodules

sarahelsaig committed:
Downgrade pnpm vesion.

Piedone commented:
Nice, I wanted to ask you to update these after Renovate again found dependency updates, thank you!

Piedone commented:
The Windows build failed.

sarahelsaig committed:
revert packageManager

Piedone committed:
Submodule src/Utilities/Lombiq.NodeJs.Extensions updated

Submodule src/Utilities/Lombiq.NodeJs.Extensions:
    > c69b57d - Remove the temporary Renovate config
    > f3b7b08 - Restoring the lock file's contentn
    > 2d55131 - Downgrade PNPM lock file back to original

Piedone committed:
Updating submodules to merged dev

[skip ci]
